src/red.ts

```diff
@@ -697,4 +696,9 @@ export const serializedRedEnvSourceText = (function redEnvFactory(blueEnv: Membr
 
     return getRedValue;
 
-}).toString();
\ No newline at end of file
+})
+.toString()
+// We cannot have 'use strict' directly in `redEnvFactory()` because bundlers and
+// minifiers may strip the directive. So, we inject 'use strict' after the function
+// is coerced to a string.
+.replace('{', `{\n'use strict'`);
```

Comment:
@caridy This was the source of sloppy-ness. Rollup removed the `'use strict'` directive from the source.

Reply:
Oh wow! Is this something you discovered? I tried to find documentation abut this behavior, but nothing too clear, do you have any? Just to know...